### DISCUSSION TITLE
Don’t ever change the UI, it’s perfect.

### DISCUSSION THREAD

**Discussion by powmeister** [ViennaRSS/vienna-rss#1798] [Ideas]
There is a desperate need to tell all devs that you don’t need to change perfection — to stop fixing things that aren’t broken. 

The UI of this app is perfect, from the *font choice, default size, layout, … Everything is perfect. Don’t mess with that kind of reliability and beauty in design. 

At a time when a lot of people are fvcking up user experience and UI just because of trends or faux-productivity, hold on dearly to what makes your app work and get love from users.

**Comment by barijaona** [comment]
I mostly agree, except for the Unified layout that seems like a hack and is not very smooth…
May be we should rewrite it on a NSCollectionView basis…

**Comment by Wellenbrett** [comment]
I have been using Vienna for decades and appreciate it very much. Compared to Netnewswire, what I appreciate about Vienna is the ability to manually sort my RSS feeds, create subfolders in the feed list, and the German language support. 

The only thing I like better about Netnewswire is the appearance of the article list and the display of the articles. For comparison, I'm posting screenshots of the Vienna feed as it appears in Vienna and as it appears in Netnewswire. I find the Netnewswire display clearer and the content easier to grasp intuitively. The Netnewswire display, for example, contains more white space: the advantages are obvious and not just a fashion trend.

Unfortunately, the article display in Vienna is not theme-capable (as far as I can tell). Netnewswire is open source, so the display could be quickly replicated. But the display of the article list in Vienna cannot be changed by users. I would very much like to see a standard display in Vienna like in Netnewswire.

Netnewswire:
<img width="1103" height="887" alt="Netnewswire" src="https://github.com/user-attachments/assets/f549c49a-1d83-43cb-a027-a8dffa9693dc" />

Vienna:
<img width="1102" height="772" alt="Vienna" src="https://github.com/user-attachments/assets/00c5e855-1da2-48f6-b6cd-fa777267c28e" />

**Reply by TAKeanice** [reply]
Hi, you can change the theme for the article display in Vienna if you don't display the whole website but only the actual RSS article.

https://www.vienna-rss.com/extras/

**Reply by Wellenbrett** [reply]
Thanks, yes, I know. I've tried all the themes, but none of them come close to Netnewswire in terms of clarity and readability. Which one would you recommend? However, it was mainly about the display of the article list—and as far as I know, users cannot change how it is displayed. I also think that the display of the article list and articles should be consistent.

**Reply by TAKeanice** [reply]
That's right, the article list itself is not adaptable. Thanks for the suggestion to enhance its appearance. I also don't think that it makes the app less usable doing so.
It's a different topic to make it consistent with the article style, since the styles are CSS which is not applicable in UIKit, so I would rather go for a generally cleaner but generic look.

**Reply by Wellenbrett** [reply]
Thank you very much for your kind words. Although I am a programmer myself, I am purely a user of Vienna. With all due respect for the work of the Vienna developers, I would like to suggest first making the article list in UIKit clearer (preferably like in Netnewswire, in my personal opinion) and then creating a CSS for the article display that aligns it with the UIKit display of the article list. Personally, I don't really need any theming in Vienna (although that is of course a feature) as long as the articles are displayed as clearly as in NetNewswire.

**Reply by TAKeanice** [reply]
> Which one would you recommend?

On this: If you want you can design or tweak a theme, and we would be happy to take it as a contribution. What exactly are you missing in the existing themes for the article view? Maybe that is worth a GitHub issue on its own :)
